frontend-react/src/utils/misc.ts

```diff
@@ -46,6 +46,38 @@ export const checkJson = (
     }
 };
 
+export function includesSpecialChars(text: string): boolean {
+    return /[^a-z\d\s]/gi.test(text);
```

Comment:
Does this need to be global?  (re: the `g` flag at the end)

Reply:
Good point since the same rules apply for a new org name in theory.

And thinking about it, maybe also Value Sets?

Reply:
No, not necessary